src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -245,6 +349,95 @@ private Exception GetCurlException(int code)
             return new Exception(GetCurlErrorString(code));
         }
 
+        private void SetCurlOption(SafeCurlHandle handle, int option, object value)
```

Comment:
I didn't realize until getting to this point that SetCurlOption takes an object as the last argument.  That's going to result a lot of unnecessary boxing.